common/services/allocation.js

```diff
@@ -14,16 +14,17 @@ const allocationService = {
     'to_location',
   ],
 
-  cancel(allocationId) {
+  cancel(data) {
```

Comment:
Could we use the same pattern here that is used to [cancel moves](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/9614729dec89d030652f2001eac27d386fbb11eb/common/services/move.js#L269-L283)  by where we keep the first argument as the ID and pass the data in as the secon?